kafka-ui-react-app/src/components/common/table/TableHeaderCell/TableHeaderCell.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { TopicColumnsToSort } from 'generated-sources';
+import StyledTableHeaderCell from 'components/common/table/TableHeaderCell/TableHeaderCell.styled';
+import cx from 'classnames';
+
+export interface TableHeaderCellProps {
+  title?: string;
+  thType?: 'primary';
```

Comment:
Why do we need it if there's only one type of header?

Reply:
in figma there is only one design style, so I've added what we have for now.